app/controllers/provider_interface/provider_interface_controller.rb

```diff
@@ -34,6 +34,7 @@ def current_provider_user
 
     def check_cookie_preferences
       @google_analytics_id = ENV.fetch('GOOGLE_ANALYTICS_MANAGE', '') if cookies['consented-to-manage-cookies'].eql?('yes')
+      @provider_type = current_provider_user && current_provider_user.primary_provider_type
```

Comment:
I think you might need to use `current_provider_user&.primary_provider_type` with the new rubocop rules (went in recently)